rsvp/views.py

```diff
@@ -37,14 +43,44 @@ def event_signup(request, event_id):
         raise PermissionDenied
     event = get_object_or_404(Event, id=event_id)
     rsvp = Reservation.objects.filter(event=event, user=user)
-    print("event_signup",len(rsvp))
+    emails = []
+    variables = {}
+    nameDict = {}
+    context = {}
+    context["name"] = user.first_name
+    emails.append(user.email)
+    nameDict["first_name"] = user.first_name
+    variables[user.email] = nameDict
+    variables = json.dumps(variables)
+    print(variables)
     if rsvp:
         rsvp_single = rsvp.first()
         rsvp_single.attending = not rsvp_single.attending
         rsvp_single.save()
+        context["event_title"] = rsvp_single.event.title
+        context["event_location"] = rsvp_single.event.location
+        context["event_datetime"] = rsvp_single.event.start_time
+        context["event_url"] = EVENT_BASE_URL + str(rsvp_single.event.slug) + '/'
+        message = get_template('rsvp/email_template.html').render(Context(context))
+        if rsvp_single.attending :
+            email = EmailMultiAlternatives('Weekly Study Meetup - ' + str(rsvp_single.event.title), '' , 'seoul@djangogirls.org', emails)
```

Comment:
Probably better to use `str.format` here instead of string concat https://docs.python.org/3/library/stdtypes.html#str.format